### DISCUSSION TITLE
After frequent manipulation with PixiJS, the image disappears, as shown below

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#8206] [Q&A]
After frequent manipulation with PixiJS, the image disappears, as shown below
Does PIXIJS have a property to tell if the image has disappeared?
![image](https://user-images.githubusercontent.com/89827437/155944382-fd13405a-9b70-4a25-9445-513a2f90a5c4.png)

**Comment by ivanpopelyshev** [comment]
Need more details.

**Reply by pcbren1987** [reply]
This is a warning
WARNING: Too many active WebGL contexts. Oldest context will be lost.
![image](https://user-images.githubusercontent.com/89827437/156097520-fc02ec8b-65a6-4caa-9e68-e85c999bb315.png)

**Comment by changeyoucant** [comment]
你好 我也遇到了 同样的问题 请问您是怎么解决的？
